scapy/contrib/gtp_v2.py

```diff
@@ -345,8 +367,8 @@ class ULI_RAI(ULI_Field):
         # MNC: if the third digit of MCC is 0xf, then the length of
         # MNC is 1 byte
         gtp.TBCDByteField("MNC", "", 1),
-        ShortField("LAC", 0),
-        ShortField("RAC", 0),
+        BitField("LAC", 0, 16),
```

Comment:
Why is this change needed?

Reply:
Same comment as https://github.com/secdev/scapy/pull/2488/files#r425166992 
Took the reference from existing field - https://github.com/secdev/scapy/blob/master/scapy/contrib/gtp_v2.py#L362